arelle/plugin/validate/EDINET/rules/upload.py

```diff
@@ -1098,6 +1099,73 @@ def rule_EC1031E(
             )
 
 
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[DISCLOSURE_SYSTEM_EDINET],
+)
+def rule_EC8023W(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    EDINET.EC8023W: In IXBRL files, 'nonFraction' elements should be immediately preceded by
+    '△' if and only if the sign attribute is '-'.
+
+    * Tagging using International Financial Reporting Standards taxonomy elements is not checked.
+    * Tagging using Japanese GAAP notes or IFRS financial statement filer-specific additional elements
+        may be identified as an exception and a warning displayed, even if the data content is correct.
+    """
+    negativeFactXpath = """
+    //ix:nonFraction[@sign="-"][
```

Comment:
Does this still work if the document uses non-standard namespace prefixes (something other than ix)? Or would we be better off defining our own namespace map for our usage of the ix prefix instead of using what's in the document?